ID_CONVEYOR_No12: 4
ID_CONVEYOR_No34: 5
ID_CONVEYOR_No56: 6
ID_MAIN         : 3
ID_HC           : 1
ID_RTC          : 2